lib/active_model/serializer.rb

```diff
@@ -21,7 +21,7 @@ class Serializer
     include Caching
     include Links
     include Type
-    require 'active_model/serializer/adapter'
+    require 'active_model_serializers/adapter'
```

Comment:
I removed the adapter dependency and added adapter method with deprecation warning. To replace `adapter` method, I created a similar one in Adapter module called `lookup_from_config`. Can you look at the changes I made  in the separate branch I made https://github.com/domitian/active_model_serializers/compare/move-namespace-of-adapter-to-active-model-serializers...domitian:removing_adapter_from_serializer#diff-5849afc4f57a4ff11910f672112e293aR16